elementary/utils/time.py

```diff
@@ -51,3 +55,33 @@ def convert_datetime_utc_str_to_timezone_str(
         )
     except Exception:
         return isoformat_datetime
+
+
+def convert_partial_utc_iso_format_to_full_iso_format(
```

Comment:
These functions are very similar.
What do you think about having an `is_utc` flag and merge them, or alternatively set default timezone to UTC if no timezone is defined?

Reply:
Sounds good.